typescript/api/services/VocabService.ts

```diff
@@ -137,6 +137,7 @@ export module Services {
         searchString = ` AND (${queryString})`;
       }
 
+      //TODO instead of call external mint perform an Ajax call to ReDBOx or directly serach in the database for records or in the solr index ???
```

Comment:
Remove the TODO. I'm expecting this method will be left untouched when we search for records within ReDBox